lib/system/zephyr/generic/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+collect (PROJECT_LIB_HEADERS sys.h)
```

Comment:
https://github.com/OpenAMP/libmetal/pull/208 probably makes this patch obsolete
I can move the` virtio_exp` branch on top of it